docs/binary.md

```diff
@@ -5,10 +5,10 @@
 You can download pre-built binaries for the Promscale
 Connector [on our release page](/releases).
```

Comment:
This link needs to be changed to https://github.com/timescale/promscale/releases as well.

Reply:
done!  :)